packages/notification-center/src/index.ts

```diff
@@ -80,6 +80,7 @@ export interface INotificationsContext {
     storeId?: string
   ) => void;
   refetch?: (storeId?: string, query?: IStoreQuery) => void;
+  disposeDefault?: () => void;
```

Comment:
I think this one is redundant :)

Reply:
@scopsy Oh my, it is!
Thanks for noticing! ♥️